.gitignore

```diff
@@ -1,6 +1,8 @@
 .*
 !.github
 !.gitignore
+!dbmongo/js/.nvmrc
```

Comment:
Ce fichier contient le numéro de version de Node.js nécessaire pour que les nouveaux tests puissent s'exécuter.

Depuis le répertoire `dbmongo/js`, il suffit de taper `nvm use` pour utiliser cette version. (voir [nvm-sh/nvm: Node Version Manager - POSIX-compliant bash script to manage multiple active node.js versions](https://github.com/nvm-sh/nvm))